README.md

```diff
@@ -1 +1,220 @@
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Space-Travelers-Hub] <a name="about-project"></a>
+
+### is a web application that provides commercial and scientific space travel services. This application will allow users to book rockets and join selected space missions
+- Reserve a Rocket
+- Have some missions
+- Check Profile
+
+## 🛠 Built With <a name="built-with"></a>
+
+- HTML
+- CSS
+- JavaScript
+- NPM
+- React
+- React router
+- Redux
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="">React</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="">External API</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="">External API</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Display Rockets, Missions pages using SpaceX API]**
+- **[Display Mission enrollment and Rocket reservation status and allow to change them]**
+- **[Display of user Profile with list of enrolled missions and reserved rockets]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+<a href="https://space-travelers-hub-mvxs.onrender.com">Live</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+
+- [ ] A PC running any operating system like Windows, Linux, or MacOS
+- [ ] An Internet connection
+- [ ] Git installed
+- [ ] Node.js
+- [ ] Knowledge of some basic Git commands
+
+### Setup
+
+- [ ] Link your PC to GitHub
+- [ ] Clone this repository to your desired folder:
+
+Clone this repository to your desired folder:
+
+```git clone https://github.com/shayan1234554321/Space-Travelers-Hub.git```
+
+- [ ] Open the file in your code editor
+
+### Install
+
+- [ ] Setup JavaScript + CSS + HTML linter
+
+Run the following codes in your project folder in your IDE:
+
+```
+npm install --save-dev eslint@7.x eslint-config-airbnb@18.x eslint-plugin-import@2.x eslint-plugin-jsx-a11y@6.x eslint-plugin-react@7.x eslint-plugin-react-hooks@4.x @babel/eslint-parser@7.x @babel/core@7.x  @babel/plugin-syntax-jsx@7.x @babel/preset-env@7.x  @babel/preset-react@7.x
+```
+
+```
+npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+```
+
+- [ ] project dependencies
+
+```
+npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+Start the server
+
+```
+npm start
```

Comment:
Kindly make sure your commands are correct. Currently, you using `npm run dev` to start the server. 👍